src/support/World.coffee

```diff
@@ -0,0 +1,20 @@
+rek = require 'rekuire'
+chai = require 'chai'
+chaiAsPromised = require 'chai-as-promised'
+ElementHelper = rek 'ElementHelper'
+Transform = rek 'Transform'
+
+class World
+  Q: require 'q'
+  elementHelper: new ElementHelper()
+  transform: new Transform()
+  currentPage: null
+  pageObjectMap: null
+
+  constructor: (callback) ->
+    chai.use chaiAsPromised
+    @expect = chai.expect
+
+    callback()
```

Comment:
I'm curious why the World constructor takes a callback if it has no async operations (or at least calls the callback synchronously).